sql/src/main/java/org/apache/druid/sql/http/ObjectWriter.java

```diff
@@ -93,4 +96,44 @@ public void close() throws IOException
   {
     jsonGenerator.close();
   }
+
+  static void writeHeader(
```

Comment:
Do we need to keep these new methods static?
If it's for testing, we could test these by calling the `writeHeader` method provided by the interface `ResultFormat.Writer`.

Same comment in other writer implementations too.

Reply:
ObjectWriter.writeHeader is static because ObjectLinesWriter also uses it; same for ArrayWriter.writeHeader and ArrayLinesWriter.